cypress/utils/dashboards/data_explorer/data_explorer_page.po.js

```diff
@@ -0,0 +1,507 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Wouldn't it be better to keep state purely within the Spec file and leave the page class stateless? We would have to create a new Page object for each language or create a setter for the language that would need to be used every time a new language is selected in the test spec otherwise. I'm open to making that change, but we can talk about this.